packages/blockchain-wallet-v4-frontend/src/middleware/analyticsMiddleware/types.ts

```diff
@@ -317,12 +338,28 @@ type LinkBankClickedPayload = BasePayload & {
   origin: LinkBankClickedOrigin
 }
 
+type LoginHelpClickedPayload = BasePayload & {
+  origin: 'PASSWORD' | 'IDENTIFIER' | 'QR_CODE'
```

Comment:
You may want to consider making all these an enum. You'll be able to reuse a lot of them and get much nicer auto complete.